sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -287,8 +287,8 @@ object ScalaNativePluginInternal {
     javacOptions ++= Seq("-source", "1.8", "-target", "1.8"),
     initialize := {
       val res = initialize.value
-      if (sys.props("java.specification.version") != "1.8")
-        sys.error("Java 8 is required for this project.")
+      if (sys.props("java.specification.version").toDouble < 1.8)
```

Comment:
Thanks, I quickly looked at `java.specification.version` but didn't find evidence that it may have a different format. I'll change that.